src/java.desktop/unix/classes/sun/awt/X11GraphicsDevice.java

```diff
@@ -105,6 +110,25 @@ public int getType() {
         return TYPE_RASTER_SCREEN;
     }
 
+    public int scaleUp(int x) {
```

Comment:
This is copied not moved from the X11GraphicsConfig, just to minimize the change in the fix. I Will clean up this later.